src/Tribe/RSVP.php

```diff
@@ -2067,4 +2071,30 @@ protected function parse_ticket_quantity( $product_id ) {
 	public function generate_security_code( $attendee_id ) {
 		return substr( md5( rand() . '_' . $attendee_id ), 0, 10 );
 	}
+
+	/**
+	 * Ensure we update the stock when deleting attendees from the admin side
+	 * @since TBD
+	 *
+	 * @param $attendee_id
+	 *
+	 * @return bool|void
+	 */
+	public function update_stock_from_attendees_page( $attendee_id ) {
+		$attendee = get_post( $attendee_id );
+
+		// Can't find the attendee post
+		if ( empty( $attendee ) || self::ATTENDEE_OBJECT !== $attendee->post_type ) {
```

Comment:
Split this into two conditionals and `return false;`

Reply:
Why split it?